src/detect-threshold.h

```diff
@@ -32,35 +32,41 @@
 #define TYPE_SUPPRESS  6
 #define TYPE_BACKOFF   7
 
-#define TRACK_DST      1
-#define TRACK_SRC      2
-#define TRACK_RULE     3
-#define TRACK_EITHER   4 /**< either src or dst: only used by suppress */
-#define TRACK_BOTH     5 /* used by rate_filter to match detections by both src and dst addresses */
-#define TRACK_FLOW     6 /**< track by flow */
+#define TRACK_DST    1
+#define TRACK_SRC    2
+#define TRACK_RULE   3
+#define TRACK_EITHER 4 /**< either src or dst: only used by suppress */
+#define TRACK_BOTH   5 /* used by rate_filter to match detections by both src and dst addresses */
+#define TRACK_FLOW   6 /**< track by flow */
 
 /* Get the new action to take */
-#define TH_ACTION_ALERT     0x01
-#define TH_ACTION_DROP      0x02
-#define TH_ACTION_PASS      0x04
-#define TH_ACTION_LOG       0x08
-#define TH_ACTION_SDROP     0x10
-#define TH_ACTION_REJECT    0x20
+#define TH_ACTION_ALERT  0x01
+#define TH_ACTION_DROP   0x02
+#define TH_ACTION_PASS   0x04
+#define TH_ACTION_LOG    0x08
+#define TH_ACTION_SDROP  0x10
+#define TH_ACTION_REJECT 0x20
+
+/* distinct counting support (for detection_filter) */
+#define DF_UNIQUE_NONE     0
+#define DF_UNIQUE_SRC_PORT 1
+#define DF_UNIQUE_DST_PORT 2
```

Comment:
Are these the result of clang-formatting? If so, could you please keep the ones not directly related to your changes in a separate commit? Helps with reviewing and with commit history. :)